### DISCUSSION TITLE
/data/cache folder and permissions

### DISCUSSION THREAD

**Discussion by dondrusco** [linkedpipes/etl#992] [Q&A]
Hi,

I successfuly imported your pipelines for NKOD.
But I'm not able to finish the process 01 - Cache External Sources

HTTP Get part finishes OK, but the Cache MDR (and all other caches) fail on non-existing directory
/data/cache/eu-mdr

such directory doesn't exists in the container
```
jura@NB-085:~/nkod/etl$ docker exec -t 2d4bca21e09d ls /data
lp-etl

```

In the execution detail is a message
LpException : Missing property: http://plugins.linkedpipes.com/ontology/l-filesToScp#userName

I can't find any clue, how to solve the issue.

Thank you

Juraj

**Comment by jakubklimek** [comment]
Hi,

well, these are two separate things, and they are not related to LP-ETL itself, but rather to a specific usage in NKOD. Can you point to the source of the pipelines and provide context of what you are trying to achieve and in which setting so that it is clear what we are discussing?

NKOD uses file system for cache, so the used paths need to exist (or LP-ETL has to have permissions to create folders).
The second error has to do with the fact that NKOD uses SCP connections to store files in the cache.

If you are working with the Czech NKOD pipelines, they are currently not ready to be run from docker containers, that is why I am asking the first question about what are you trying to achieve.
